command/server.go

```diff
@@ -152,7 +152,7 @@ func (c *ServerCommand) Flags() *FlagSets {
 	f.StringVar(&StringVar{
 		Name:       "log-level",
 		Target:     &c.flagLogLevel,
-		Default:    "info",
+		Default:    "(not set)",
```

Comment:
I'd recommend defining a constant for this since it is coupled to logic below further down.

Reply:
I *knew* someone was going to point that out.